guides/common/modules/proc_updating-an-scc-account.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Updating_an_SCC_Account_{context}"]
+= Updating an SCC Account
+
+Use the following procedure to update your SCC account in {Project}.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-Updating_an_SCC_Account_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *SUSE Subscriptions*.
+. Select your SCC account.
+. Update your SCC account as required.
++
+For more information, see xref:Adding_an_SCC_Account_to_Server_{context}[].
+. Click *Submit* to update your SCC account.
+
+[id="cli-Updating_an_SCC_Account_{context}"]
+.CLI procedure
+* Update your SCC account in {Project}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hammer scc_manager scc_accounts update \
+--id _My_SCC_Account_ID_ \
+--location-id _My_Location_ID_ \
```

Comment:
As the scc_account is bound to a location/organization already and the scc_account_id is set, location / organization is not necessary to be set. These options will be removed soon-ish.

Reply:
Applied.